discovery-provider/src/queries/get_skipped_transactions.py

```diff
@@ -126,3 +131,20 @@ def set_indexing_error(
 
 def clear_indexing_error(redis_instance):
     redis_instance.delete(INDEXING_ERROR_KEY)
+
+
+def save_and_get_skip_tx_hash(session, redis):
+    """Fetch if there is a tx_hash to be skipped because of continuous errors
+    """
+    indexing_error = get_indexing_error(redis)
+    if isinstance(indexing_error, dict) and 'has_consensus' in indexing_error and indexing_error['has_consensus']:
+        num_skipped_tx = session.query(func.count(SkippedTransaction.id)).scalar()
```

Comment:
confirming / for my learning - do we clear the skipped transaction table every time we restart indexing?

Reply:
nope it stays there, we can truncate to clear it up but i think it's good to persist this and have a limit on how much we skip

Reply:
ooh but then if the table fills up over time what will we do after we've hit 10 errored txs? say we fix the error and restart indexing, won't any incoming breaking tx block after break indexing and the 10 limit won't apply anymore?